code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1152,13 +1153,13 @@
 	if((temp_turf.z != our_turf.z) || M.stat!=CONSCIOUS) //Not on the same zlevel as us or they're dead.
 //		to_chat(world, "[(temp_turf.z != our_turf.z) ? "not on the same zlevel as [M]" : "[M] is not concious"]")
 		if(temp_turf.z != 2)
-			to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
-
-
+			if(prob(5))//Can't tell who is missing as easily as before. I feel like this is a fun alternative to removing it altogether
```

Comment:
3. Give future coders a headache trying to recreate a bug with no obvious steps or cause whenever it happens to a user.

And that doesn't explain why the probability is there if they're not meant to know AT ALL.